regionmask/tests/test_unstructured.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+import pytest
+import xarray as xr
+
+from .utils import dummy_region, expected_mask_2D
+
+
+@pytest.mark.parametrize("method", ["shapely", "pygeos"])
+def test_unstructured_dummy(method):
+    """Test for unstructured output."""
+    lat = [0.5, 0.5, 1.5, 1.5]
+    lon = [0.5, 1.5, 0.5, 1.5]
+    grid = xr.Dataset(coords={"lon": ("cells", lon), "lat": ("cells", lat)})
+
+    result = dummy_region.mask(grid, method=method, wrap_lon=False)
+
+    expected = expected_mask_2D().flatten()
+    assert isinstance(result, xr.DataArray)
+    assert np.allclose(result, expected, equal_nan=True)
+    for c in ["lon", "lat"]:
+        assert np.all(np.equal(grid.coords[c], result.coords[c]))
```

Comment:
hope this coords test is sufficient. cannot compare to `dummy_lat` and `dummy_lon` because different shape/size